udp-ballerina/connectionless_client.bal

```diff
@@ -46,11 +46,11 @@ public client class Client {
 
     # Reads data from the remote host. 
     # ```ballerina
-    # Datagram|ReadTimedOutError result = socketClient->receive();
+    # udp:Datagram|udp:Error? result = socketClient->receive();
```

Comment:
```suggestion
    # udp:Datagram|udp:Error? result = socketClient->receiveDatagram();
```
Method names has to be updated according to the design doc.

Reply:
updated